ts/components/services/ServiceSectionListComponent.tsx

```diff
@@ -38,13 +40,33 @@ const styles = StyleSheet.create({
  * A component to render a list of services grouped by organization.
  */
 class ServiceSectionListComponent extends React.Component<Props> {
+  private isRead = (
+    potService: pot.Pot<ServicePublic, Error>,
+    readServices: ServicesByReadState
+  ): boolean => {
+    const service =
+      pot.isLoading(potService) ||
+      pot.isError(potService) ||
+      pot.isNone(potService)
+        ? undefined
+        : potService.value;
+
+    return (
+      readServices !== undefined &&
+      Object.keys(readServices).find(k => {
```

Comment:
readServices is an object with serviceId as key, you don't need to create an array.

Reply:
Fix in 83181dc